website/content/blog/2024-09-11-meeting.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Board Meeting Minutes
+date: "2024-09-11"
+excerpt: The year in review
+tags: foundation
+author: MonoGame Foundation
+image: /images/blog/cover/monogame_foundation.png
+ogImage: /images/blog/cover/monogame_foundation.png
+---
+
+## Attendee's
+
+- Dean Ellis - President
+- Dominique Louis - Treasurer
+- Simon Jackson - Corporate Secretary
+- Tom Spilman - Banking Matters
+- Marko Jeremic - Board Chairman
+- Thomas Altenburger - Board Member
+
+## Agenda
+
+- Content Pipeline enhancements.
+- Release improvements.
+- Bounties updates.
+- Full-time appointment.
+- Releases are go.
+
+## Content Pipeline enhancements.
+
+A common thread we come across from veterans or users of other tools focuses on the Content Pipeline, basically "Why do I need this thing at all". While we have tried to remain informative as to the benefits of preparing, compressing, and formatting content in advance for platforms, there are still those who would prefer not to.
+
+So, in the spirit of the Foundation's desire to keep things moving, we have taken the bold decision to improve direct access to content for those that need it, and rather than force users (as was always the case in the past) to duplicate code or implement complicated strategies for handing content, we have kicked off an initiative to simplify content loading from files by hiding the complexity behind the very same methods used to load content from the content pipeline.

```

Comment:
Nitpicking but this sentence sounds a bit presumptuous to me. I mean, we didn't invent anything here nor made this bold of a move.

Reply:
Yeah this is way too much....  it can be two sentences:

> We discussed and decided to allow loading of raw content files (pngs, wav, etc) via `ContentManager` as a fallback when an XNB is not found.  This makes things nicer for users that dislike pre-compiling content, but allows you to change that decision later without rewriting your game code.